updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,171 @@
+:_content-type: ASSEMBLY
+[id="mirroring-ocp-image-repository"]
+= Mirroring the OpenShift Container Platform image repository
+include::_attributes/common-attributes.adoc[]
+:context: mirroring-ocp-image-repository
+
+toc::[]
+
+You must mirror container images onto a mirror registry before you can update a cluster in a disconnected environment. You can also use this procedure in connected environments to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+[NOTE]
+====
+This registry must be running at all times as long as the cluster is running.
+====
+
+There are two supported methods for mirroring images onto a mirror registry:
+
+* Using the oc-mirror OpenShift CLI (`oc`) plugin
```

Comment:
I'm not sure if it's appropriate right here or not (because you'd need people do to the "preparing your host" part), but could consider linking down to these sections

Reply:
That's a good suggestion. I guess if I ensure that both mirroring sections have explicit prereqs that mention the processes you would perform in the "preparing your host section", the usefulness of the links might be worth accidentally leading users to skip the preparing section on occasion. 

Unlike the [prereqs of the oc adm release mirror section](https://55448--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/mirroring-image-repository.html#update-mirror-repository-adm-release-mirror_mirroring-ocp-image-repository), there's no prereq in the oc mirror section that mentions anything about downloading a pull secret (which would be done in the procedure for "Configuring credentials that allow images to be mirrored"). Would you mind if I put this prereq somewhere in your oc mirror modules and made sure with an SME/QE that it's a valid addition?

Reply:
Yes I think that would make sense - to list the prereqs in that preparing section, so that if people do skip it, they see it later on there.